docs/contributors/triage.md

```diff
@@ -52,7 +52,14 @@ Generally speaking, the following labels are very useful for triaging issues and
 * Needs Testing - when old bugs seem like they are no longer relevant.
 * [Type] Help Request - when someone is asking for general help with setup/implementation. 
 
-## Design specific triage
+### Release specific triage
+Here are some guidelines to follow when doing triage specifically around the time of a release. This is important to differentiate compared to general triage so problematic, release blocking bugs are properly identified and solutions are found.
+
+* If a bug is introduced in a release candidate (RC) and it's going to break many workflows, add it to the RC milestone and flag in the #core-editor channel in WordPress.org slack.
```

Comment:
While the general risk is low and it's likely to be understood easily enough, the phrasing "RC milestone" could be potentially confusing. The milestone isn't related to the release candidate specifically, but instead for the version of the plugin to which the release candidate corresponds. There is no "Gutenberg 8.1 RC" milestone, but there is a "Gutenberg 8.1" milestone. It'd probably be clearer to reference it as the milestone for the version, in case someone might be confused by the lack of a release candidate milestone.

```suggestion
* If a bug is introduced in a release candidate (RC) and it's going to break many workflows, add it to the version milestone and flag in the #core-editor channel in WordPress.org slack.
```

(Or some variation, e.g. "milestone for the version", "version milestone associated with the release candidate") 

There's a few other occurrences of "RC milestone" in the following points as well, for which the same would apply.